packages/tools/devtools/devtools-example/package.json

```diff
@@ -89,12 +98,15 @@
 		"eslint-plugin-react-hooks": "~4.6.0",
 		"html-webpack-plugin": "^5.5.0",
 		"jest": "^29.6.2",
+		"jest-dev-server": "^4.3.0",
 		"jest-environment-jsdom": "^29.6.2",
 		"jest-junit": "^10.0.0",
 		"pm2": "^5.3.0",
 		"prettier": "~2.6.2",
+		"puppeteer": "^17.1.3",
 		"rimraf": "^4.4.0",
-		"start-server-and-test": "^1.11.7",
+		"sinon": "^7.4.2",
```

Comment:
Are we using sinon / sinon-chrome for anything? If not, let's remove these and the corresponding @types imports

Reply:
Cleaned up